bentoml/_internal/yatai_client/__init__.py

```diff
@@ -245,9 +246,9 @@ def push_model(model: "Model"):
                 runnable_type=r.runnable_type,
                 models=r.models,
                 resource_config=BentoRunnerResourceSchema(
-                    cpu=r.resource_config.cpu,
-                    nvidia_gpu=r.resource_config.nvidia_gpu,
-                    custom_resources=r.resource_config.custom_resources,
+                    cpu=r.resource_config.get("cpu"),
```

Comment:
I hope this type-level refactoring needs to be synchronized to all the places where the type is used, and it feels like we need to add all the type checks to our ci, otherwise, it's a nullity